pkgs/development/interpreters/supercollider/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, stdenv, mkDerivation, fetchurl, cmake
+{ lib, stdenv, mkDerivation, fetchurl, cmake, bash
```

```suggestion
{ lib, stdenv, mkDerivation, fetchurl, cmake, runtimeShell
```

Arg fixup for above review item